groups-domains.php

```diff
@@ -48,7 +48,8 @@
                                 <div class="col-md-6">
                                     <div class="form-group">
                                         <label for="new_domain">Domain:</label>
-                                            <input id="new_domain" type="url" class="form-control active" placeholder="Domain to be added" autocomplete="off" spellcheck="false" autocapitalize="none" autocorrect="off">
+                                        <input id="new_domain" type="url" class="form-control active" placeholder="Domain to be added" autocomplete="off" spellcheck="false" autocapitalize="none" autocorrect="off">
+                                        <div id="suggest_domains" style="display: none; overflow-x: auto;"></div>
```

Comment:
Please, remove the inline style 
```css
style="display: none; overflow-x: auto;"
```
and move it into `styles/pi-hole.css`.

Reply:
In the latest commit I've managed to use existing classes to get the desired behavior.